timetracker/entries/migrations/0004_auto_20150910_0348.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('entries', '0003_create_fk_to_client'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='client',
+            name='author',
+            field=models.ForeignKey(to=settings.AUTH_USER_MODEL, default=None),
```

Comment:
E.g. fall back to first superuser would be a possible fallback, but I don't like it.